integrations/observability/nginx/getting-started/opensearch_dashboards.yml

```diff
@@ -0,0 +1,368 @@
+# OpenSearch Dashboards is served by a back end server. This setting specifies the port to use.
```

Comment:
For readability, we don't need to duplicate the entire OSD config documentation here -- let's delete all the comments and only leave the config relevant for this specific scenario.